src/main/java/stream/flarebot/flarebot/util/WebUtils.java

```diff
@@ -77,7 +83,7 @@ public static boolean pingHost(String host, int timeout) {
         return pingHost(host, 80, timeout);
     }
 
-    public static boolean pingHost(String host, int port, int timeout) {
+    private static boolean pingHost(String host, int port, int timeout) {
```

Comment:
Is this not used anywhere else? 👀 I thought it was...

Reply:
Nope